frontend/src/routes/Search.tsx

```diff
@@ -585,7 +599,145 @@ const SearchSeries: React.FC<SearchSeriesProps> = ({ id, title, description, hos
                 />}
             </div>
         </WithIcon>
-    </Item>;
+        <ThumbnailStack {...{ thumbnailInfo, title }} />
+    </Item>
+;
+
+type ThumbnailStackProps = {
+    title: string;
+    thumbnailInfo: readonly {
+        readonly audioOnly: boolean;
+        readonly isLive: boolean;
+        readonly thumbnail: string | null | undefined;
+    }[] | undefined;
```

Comment:
This type definition is duplicated. Try `SearchSeriesProps["thumbnailInfo"]` or even set `type ThumbnailStackProps = Pick<SearchSeriesProps, "title" | "thumbnailInfo">`